tide_core.module

```diff
@@ -181,6 +203,25 @@ function tide_core_form_user_register_form_alter(&$form, FormStateInterface $for
   if (isset($form['account']['mail']['#required'])) {
     $form['account']['mail']['#required'] = TRUE;
   }
+  $form['account']['name']['#weight'] = 2;
```

Comment:
Hi @MdNadimHossain 
This is the new code for re-order the fields.
Thanks

Reply:
same for this one - why we are manipulating the weight in the alter hook, why can't we just make the changes in the config file directly using the update hook and then export the settings ?

Reply:
Hi @MdNadimHossain 

The fields are 'based fields', defined by code. they can't configure the form display setting.
<img width="1718" alt="Screen Shot 2022-11-10 at 1 42 49 pm" src="https://user-images.githubusercontent.com/6699227/200987887-0e578f1e-6fac-4db2-ad23-86f929adee1a.png">
The fields are grouped in side $form['account'] fieldset, custom field can't be config to show in the field set by UI.
The 'status', 'roles', 'notify', and 'password_policy_status' fields even doesn't show in the config form setting.
The 'notify' and 'password_policy_status' fields added by the contributor module, they not used on all sites so we need to use a code to check if the field exists.
That is why It needs code to set correctly order display.

Thanks